packages/std/src/mock.rs

```diff
@@ -235,39 +235,68 @@ fn mock_ibc_channel(my_channel_id: &str, order: IbcOrder, version: &str) -> IbcC
         },
         order,
         version: version.to_string(),
-        counterparty_version: Some(version.to_string()),
         connection_id: "connection-2".to_string(),
     }
 }
 
 #[cfg(feature = "stargate")]
-/// Creates a IbcChannelOpenMsg for testing ibc_channel_open.
-pub fn mock_ibc_channel_open(
+/// Creates a IbcChannelOpenMsg::OpenInit for testing ibc_channel_open.
+pub fn mock_ibc_channel_open_init(
     my_channel_id: &str,
     order: IbcOrder,
     version: &str,
 ) -> IbcChannelOpenMsg {
-    IbcChannelOpenMsg::new(mock_ibc_channel(my_channel_id, order, version))
+    IbcChannelOpenMsg::new_init(mock_ibc_channel(my_channel_id, order, version))
+}
+
+#[cfg(feature = "stargate")]
+/// Creates a IbcChannelOpenMsg::OpenTry for testing ibc_channel_open.
+pub fn mock_ibc_channel_open_try(
```

Comment:
This distinction (6 different mock functions) is what I'm not sure about. It was the simplest thing to do and it seems to work alright for `ibc-reflect` and `ibc-reflect-send` tests. Thoughts?

Reply:
Looks good to me.
There are 6 cases for messages now, even if there are only 3 entry points, we simply grouped them into pairs (very explicitly with enums)

Reply:
Nice. Better to have all 6 than complicated ways around it.